### ISSUE TITLE
2021.2.5f1c1 版本依赖package之后打开crash

### ISSUE THREAD

**Issue by JerryJson** [Unity-Technologies/com.unity.uiwidgets#329]
Identifier:            com.unity3d.UnityEditor5.x
Version:               Unity version 2021.2.5f1c1 (2021.2.5f1c1)

Thread 0 Crashed:: tid_103 Dispatch queue: com.apple.main-thread
0   libUIWidgets.dylib            	       0x15bb1f0f7 uiwidgets::UIWidgetsSystem::BindUnityInterfaces(IUnityInterfaces*) + 39
1   Unity                         	       0x1038bca64 RegisterPlugin(void*, UnityPluginFunctions&) + 580
2   Unity                         	       0x1038bccdd InitializePlugin(void*) + 173
3   Unity                         	       0x1038be84f FindAndLoadUnityPlugin(char const*, void**, bool) + 207
4   libmonobdwgc-2.0.dylib        	       0x14e5ec7ec lookup_pinvoke_call_impl + 704
5   libmonobdwgc-2.0.dylib        	       0x14e5ed2f6 mono_lookup_pinvoke_call_internal + 34
6   libmonobdwgc-2.0.dylib        	       0x14e3e7d41 mono_jit_compile_method_with_opt + 913
7   libmonobdwgc-2.0.dylib        	       0x14e46f49a common_call_trampoline + 1382
8   libmonobdwgc-2.0.dylib        	       0x14e46eee3 mono_magic_trampoline + 106
9   ???                           	       0x10b199396 ???
10  ???                           	       0x1526e6af3 ???
11  ???                           	       0x1526e6e7c ???
12  libmonobdwgc-2.0.dylib        	       0x14e3ec07c mono_jit_runtime_invoke + 1587
13  libmonobdwgc-2.0.dylib        	       0x14e5a14d9 do_runtime_invoke + 80
14  libmonobdwgc-2.0.dylib        	       0x14e5a4cf6 mono_runtime_try_invoke_array + 1164
15  libmonobdwgc-2.0.dylib        	       0x14e550310 ves_icall_InternalInvoke + 877
16  libmonobdwgc-2.0.dylib        	       0x14e560d07 ves_icall_InternalInvoke_raw + 102
17  ???                           	       0x1526e6636 ???
18  ???                           	       0x1526e5d7b ???
19  ???                           	       0x1425897be ???
20  libmonobdwgc-2.0.dylib        	       0x14e3ec07c mono_jit_runtime_invoke + 1587
21  libmonobdwgc-2.0.dylib        	       0x14e5a14d9 do_runtime_invoke + 80
22  libmonobdwgc-2.0.dylib        	       0x14e5a1416 mono_runtime_invoke + 94
23  Unity                         	       0x103c6f847 scripting_method_invoke(ScriptingMethodPtr, ScriptingObjectPtr, ScriptingArguments&, ScriptingExceptionPtr*, bool) + 71
24  Unity                         	       0x103c6a7d6 ScriptingInvocation::Invoke(ScriptingExceptionPtr*, bool) + 166
25  Unity                         	       0x103d6aee6 Scripting::UnityEditor::EditorAssembliesProxy::ProcessInitializeOnLoadMethodAttributes(ScriptingExceptionPtr*) + 166
26  Unity                         	       0x103c18339 MonoManager::SetupLoadedEditorAssemblies(dynamic_array<int, 0ul> const&) + 1593
27  Unity                         	       0x103c1a888 MonoManager::EndReloadAssembly(DomainReloadingData&, dynamic_bitset) + 1432
28  Unity                         	       0x103c189b6 MonoManager::ReloadAssembly() + 310
29  Unity                         	       0x1048360ea LoadDomainAndUserAssemblies() + 58
30  Unity                         	       0x10483559a LoadInitAssemblies(LoadAssemblies, bool) + 666
31  Unity                         	       0x1050d53e8 RefreshInternalV2(AssetDatabase::UpdateAssetOptions, ScanFilter const&, InternalRefreshFlagsV2) + 16760
32  Unity                         	       0x1050ada23 StopAssetImportingV2Internal(AssetDatabase::UpdateAssetOptions, InternalRefreshFlagsV2, ScanFilter const*, char const*) + 1987
33  Unity                         	       0x1050afa2c InitialRefreshV2(bool) + 60
34  Unity                         	       0x104c0a405 Application::InitializeProject() + 17589
35  Unity                         	       0x105b25eac -[EditorApplication applicationDidFinishLaunching:] + 140
36  CoreFoundation                	    0x7ff815d7be73 __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 12
37  CoreFoundation                	    0x7ff815e19349 ___CFXRegistrationPost_block_invoke + 49
38  CoreFoundation                	    0x7ff815e192c6 _CFXRegistrationPost + 496
39  CoreFoundation                	    0x7ff815d4d786 _CFXNotificationPost + 733
40  Foundation                    	    0x7ff816b95996 -[NSNotificationCenter postNotificationName:object:userInfo:] + 82
41  AppKit                        	    0x7ff8187b818b -[NSApplication _postDidFinishNotification] + 305
42  AppKit                        	    0x7ff8187b7edd -[NSApplication _sendFinishLaunchingNotification] + 208
43  AppKit                        	    0x7ff8187b5ab0 -[NSApplication(NSAppleEventHandling) _handleAEOpenEvent:] + 541
44  AppKit                        	    0x7ff8187b5707 -[NSApplication(NSAppleEventHandling) _handleCoreEvent:withReplyEvent:] + 665
45  Foundation                    	    0x7ff816bc1284 -[NSAppleEventManager dispatchRawAppleEvent:withRawReply:handlerRefCon:] + 308
46  Foundation                    	    0x7ff816bc10f6 _NSAppleEventManagerGenericHandler + 80
47  AE                            	    0x7ff81c402d28 0x7ff81c3f7000 + 48424
48  AE                            	    0x7ff81c402592 0x7ff81c3f7000 + 46482
49  AE                            	    0x7ff81c3fb9c7 aeProcessAppleEvent + 419
50  HIToolbox                     	    0x7ff81e9b3b42 AEProcessAppleEvent + 54
51  AppKit                        	    0x7ff8187afd8a _DPSNextEvent + 2064
52  AppKit                        	    0x7ff8187adf5c -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1411
53  AppKit                        	    0x7ff8187a0359 -[NSApplication run] + 586
54  AppKit                        	    0x7ff8187741f8 NSApplicationMain + 816
55  Unity                         	       0x105b24135 EditorMain(int, char const**) + 949
56  Unity                         	       0x105b24459 main + 9
57  dyld                          	       0x111d9d4fe start + 462

**Comment by zhuxingwei** [comment]
Hi Thanks for the report !

UIWidgets 2.0 doesn't support Unity 2021 yet. Please try using Unity 2019, 2020 LTS instead. Thanks !

**Comment by JerryJson** [comment]
好的，谢谢。有计划2.0版本会适配2021或者2022LTS吗？

**Comment by zhuxingwei** [comment]
Yes we are already working on support 2021 CN LTS early this March. However, the schedule has been delayed a bit by the covid-19 issue. I will ping you in this thread when the support is done.

Since 2022 LTS will be released next year, we don't have a plan to support it for now.

**Comment by JerryJson** [comment]
OK. Thanks.
